csm/core/services/storage_capacity.py

```diff
@@ -167,10 +167,10 @@ async def get_usage(self, resource, resource_id):
 
     async def _get_user_usage(self, **request_body):
         plugin_response = await self._s3_iam_plugin.execute(const.GET_USER_CAPACITY_OPERATION, **request_body)
+        if isinstance(plugin_response, RgwError):
```

Comment:
Please add a comment that in case of an error below lines will not be executed. There is also a flaw -> create should only create the error response and return it. 
Errors should not be raised by generic classes. It should only be raised by controllers

Reply:
we can discuss this.